frontend/src/main/pages/PlayPage.js

```diff
@@ -143,7 +128,7 @@ export default function PlayPage() {
           <br />
           {!!userCommons &&
             <CardGroup >
-              <ManageCows userCommons={userCommons} commons={commons} onBuy={onBuy} onSell={onSell} />
+              <ManageCows userCommons={userCommons} onBuy={onBuy} onSell={onSell} />
```

Comment:
I think you still need `commons={commonsPlus.commons}` here... now market cow price isn't being shown

Reply:
Thanks for catching that